tests/interfaces-network/task.yaml

```diff
@@ -0,0 +1,85 @@
+summary: |
+  The network interface allows a snap to access the network as a client.
+
+  A snap which defines the network plug must be shown in the interfaces list.
+  The plug must be autoconnected on install and, as usual, must be able to be
+  reconnected.
+
+  A snap declaring a plug on this interface must be able to access network services.
+
+environment:
+  SNAP_NAME: network-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  PLUG: network
+  PORT: 8081
+  REQUEST_FILE: "./request.txt"
+  SERVICE_FILE: "./service.sh"
+  SERVICE_NAME: "test-service"
+
+kill-wait: 1m
```

Comment:
Let's please not have 1m kill timeout on anything unless it's extremely trivial. Otherwise it's way too short.. any sort of network instability would kill the test, as has happened before in the other case we discussed yesterday.

Also note that the field is `kill-timeout` not `kill-wait`.